src/games/strategy/triplea/printgenerator/PUChart.java

```diff
@@ -22,22 +22,19 @@
 import games.strategy.util.IntegerMap;
 
 class PUChart {
-  private final GameData m_data;
   private final Iterator<PlayerID> m_playerIterator;
   private final IntegerMap<PlayerID> m_moneyMap;
   private final int m_numPlayers;
   private final PlayerID[] m_playerArray;
   private final Integer[] m_moneyArray;
   private final Map<Integer, Integer> m_avoidMap;
-  private final int X_DIMENSION = 7;
-  private final int Y_DIMENSION = 6;
```

Comment:
Full code context:
``` 
   int yDimension = 6;
    int xDimension = 7;
    final int numChartsNeeded = (int) Math.ceil(((double) m_moneyMap.totalValues()) / (xDimension * yDimension));
    for (int i = 0; i < numChartsNeeded; i++) {
      m_g2d.setColor(Color.black);
      // Draw Country Names
      for (int z = 0; z < m_playerArray.length; z++) {
        final int valMod42 = m_moneyArray[z] % 42;
        final int valModXDim = valMod42 % xDimension;
        final int valFloorXDim = valMod42 / xDimension;
        if (m_avoidMap.containsKey(z) && m_moneyArray[z] / 42 == i) {
          final FontMetrics metrics = m_g2d.getFontMetrics();
          final int width = metrics.stringWidth(m_playerArray[z].getName()) / 2;
          m_g2d.drawString(m_playerArray[z].getName(), 42 + 87 * valModXDim - width, 63 + 87 * valFloorXDim);
        } else if (m_avoidMap.containsValue(z) && m_moneyArray[z] / 42 == i) {
          final FontMetrics metrics = m_g2d.getFontMetrics();
          final int width = metrics.stringWidth(m_playerArray[z].getName()) / 2;
          m_g2d.drawString(m_playerArray[z].getName(), 42 + 87 * valModXDim - width, 30 + 87 * valFloorXDim);
        } else if (m_moneyArray[z] / 42 == i) {
          final FontMetrics metrics = m_g2d.getFontMetrics();
          final int width = metrics.stringWidth(m_playerArray[z].getName()) / 2;
          m_g2d.drawString(m_playerArray[z].getName(), 42 + 87 * valModXDim - width, 60 + 87 * valFloorXDim);
        }
      }
```

I believe some of the classes in printgenerator are likely not needed, there are a couple map export functions out there that are not used.

For now fixed the variable name. There are many magic numbers going on in this method. If we wind up fixing things and not chopping off this code, at least the mess will be in just one method.